xla/service/gpu/gemm_thunk.h

```diff
@@ -38,6 +39,8 @@ class GemmThunk : public Thunk {
   GemmThunk& operator=(const GemmThunk&) = delete;
 
   Status ExecuteOnStream(const ExecuteParams& params) override;
+  Status Initialize(const GpuExecutable& /*executable*/,
```

Comment:
Nit: why have the variable name  in comments?

Reply:
Amended.